xios_examples/read_unstructured_domain_resample/test_resample_cases.py

```diff
@@ -0,0 +1,55 @@
+import copy
+import glob
+import netCDF4
+import numpy as np
+import os
+import subprocess
+import unittest
+
+import xios_examples.shared_testing as xshared
+
+this_path = os.path.realpath(__file__)
+this_dir = os.path.dirname(this_path)
+
+class TestResampleDomain(xshared._TestCase):
+    test_dir = this_dir
+    transient_inputs = ['domain_input.nc', 'domain_input_ugrid.nc']
+    transient_outputs = ['domain_output.nc', 'domain_output_ugrid.nc']
+    rtol = 5e-03
+    mesh_file = 'mesh_C12.nc'
+
+
+# A list of input `.cdl` files where XIOS is known to produce different
```

Comment:
just a comment inconsistency, there are no `input .cdl files` here, i think, they are generated from Python code

Reply:
I've changed this to not refer to .cdl files but I don't really understand what this comment means, there is no list here